hotkey.js

```diff
@@ -14,10 +14,62 @@ function ignoreHotkeysIn(target) {
   )
 }
 
+function isElementIsIgnoredInRange(parent, node) {
+  if (node.tagName === 'BODY') return null
+  if (parent === node) return null
+  if (node.hasAttribute('data-keyux-ignore-hotkeys')) return node
+
+  return isElementIsIgnoredInRange(parent, node.parentNode)
+}
+
+function findNonIgnoredElement(activeElement, elements) {
+  for (let element of elements) {
+    if (isElementIsIgnoredInRange(activeElement, element)) continue
```

Comment:
Can we move `isElementIsIgnoredInRange` content here? A small size optimization

Reply:
Sounds confusing. We need recursive calls to traverse tree.

Reply:
Got it. Sure.